app/app.go

```diff
@@ -62,7 +75,7 @@ func Run(cfg *config.Config) error {
 	// Initialize the database
 	err := db.Initialize()
 	if err != nil {
-		return fmt.Errorf("Unable to initilize the database - %s", err)
+		return fmt.Errorf("%s - %s", unableInitDB, err)
```

Comment:
Same question here, do we need this to be a variable when it's only used once?

Reply:
i tried to have all the errors together , that was all , but there is no other benefit to have it separate